packages/ndla-licenses/src/licenses.ts

```diff
@@ -9,9 +9,9 @@
 import { BY, SA, NC, ND, PD, CC0, COPYRIGHTED, CC, NA } from './licenseRights';
 import { getLocaleOrDefault, LicenseType, Locale } from './types';
 
-const freeUseNB = 'Fri gjenbruk';
-const freeUseNN = 'Fri gjenbruk';
-const freeUseEN = 'Free reuse';
+const freeUseNB = 'Offentlig eie';
+const freeUseNN = 'Offentleg eie';
```

Comment:
trur dette blir korrekt.
```suggestion
const freeUseNN = 'Offentleg eige';
```